package.json

```diff
@@ -38,7 +38,7 @@
     "travisDeployKey": "./bin/generate_travis_deploy_key"
   },
   "dependencies": {
-    "cozy-client": "27.26.3",
+    "cozy-client": "^34.2.0",
```

Comment:
I though we should remove this direct dep and use cozy-client from konnector-libs?

Reply:
You right, as i mentionned in the comment, that was intended for me to do it today :)

Reply:
This commit should be removed at all